### ISSUE TITLE
How to properly include images from _static directory in ..raw:: html on RTD?

### ISSUE THREAD

**Issue by Jacobfaib** [readthedocs/readthedocs.org#7854]
## Details
**TL;DR:**
Images links in `..raw:: html` not being rendered on RTD build due to path to image being improperly expanded/set. Should this path be relative to the file containing `..raw:: html`? Or should it be relative to the file `.. include::`'ing the html file? Or am I barking up the wrong tree?

My project is using custom css to make a pretty image grid of active team-members in raw html. This is included using `..include::` directive in Sphinx. The core html for each image used for this is:
```
<div>
  <a href='LINK/TO/GITLAB/PROFILE'><img src='LINK/TO/IMAGE' class='avatar' /></a> <br />
  <p>NAME</p>
</div>
```
`LINK/TO/IMAGE` is normally a link to their gitlab profile picture (scraped via gitlab API). For these people _everything works_  and the html looks like so:
```
<div>
 <a href='https://gitlab.com/Jfaibussowitsch'><img src='https://assets.gitlab-static.net/uploads/-/system/user/avatar/4473654/avatar.png' class='avatar' /></a> <br />
 <p>Jacob Faibussowitsch</p>
 </div>
```
But some devs don't have profile pictures. For this reason we have local images included in `_static/images`. _These do not show_, as the html looks like so:
```
<div>
 <a href='LINK/TO/GITLAB/PROFILE'><img src='/home/docs/checkouts/readthedocs.org/user_builds/petsc/checkouts/jacobf-2020-12-14-docs-port-website-to-sphinx/src/docs/sphinx_docs/_static/images/some_dev.jpg' class='avatar' /></a> <br />
 <p>NAME</p>
</div>
```
I set the `img src` path for these devs as follows in `genteamtable.py`:
```
imDir = os.path.join("..", "_static", "images")   # ../_static/images
avatar_url = os.path.join(imDir, "some_dev.gif")  # ../_static/images/some_dev.gif
```
but as you can see, on RTD it is still expanded to the path of the build rather than relative path.

The directory structure is as follows:
```
sphinx_docs/
├── conf.py
├── genteamtable.py <---------- creates table[1,2].inc
├── generated/
|   ├── table1.inc
|   └── table2.inc
├── _static/
│   └── images/
│     ├── some_dev.jpg
|     └── some_dev2.jpg
└── contact/
    └── petsc-team.rst <------- ..include:: ../generated/table[1,2].inc
```

* Read the Docs project URL: https://readthedocs.org/projects/petsc/
* Build URL (if applicable): https://readthedocs.org/projects/petsc/builds/12818647/
* Read the Docs username (if applicable): N/A?

## Expected Result
<img width="777" alt="image" src="https://user-images.githubusercontent.com/24443208/105371616-6458e180-5bca-11eb-8c45-58e8b0ec82da.png">
*A description of what you wanted to happen*

## Actual Result
Broken images for people with local images.

https://docs.petsc.org/en/jacobf-2020-12-14-docs-port-website-to-sphinx/contact/petsc_team/
*A description of what actually happened*

**Comment by stsewd** [comment]
Does this work when building locally?

**Comment by Jacobfaib** [comment]
Yes, the attached image under expected result is a local build.

**Comment by stsewd** [comment]
@Jacobfaib what's the path that the local build shows?

**Comment by Jacobfaib** [comment]
Never mind, the path I put is wrong for RTD, it is working now! Originally I had 
```
../_static/images
```
but it should have been
```
../../_static/images
```
# Solution 
All pages on RTD are treated as a directory? So while locally the URL might be
```
file:///Users/jacobfaibussowitsch/NoSync/petsc/src/docs/sphinx_docs/_build/html/contact/petsc_team.html
```
on RTD the url will be
```
https://docs.petsc.org/en/jacobf-2020-12-14-docs-port-website-to-sphinx/contact/petsc_team/
```
notice trailing slash and no `.html` suffix

**Comment by Jacobfaib** [comment]
@stsewd this may be out of scope/purview of this issue but what is the best way to detect this (ideally from within python)? Currently there are 2 ways that `genteamtable.py` is called, either locally via `makefile` (where the common command is `make html`) or how RTD does, via `conf.py`, so I'd like to have a check in there since it is the choke-point.

**Comment by stsewd** [comment]
@Jacobfaib you can access the builder name with `app.builder.name` from the sphinx extension. Also, I'd recommend using the same builder on rtd and locally to avoid any problem, thus by changing the rtd config file to use `html` or by using `make dirhtml` locally.
